css/components/_search-section.scss

```diff
@@ -80,4 +79,52 @@
             }
         }
     }
+
+    &__fake {
```

Comment:
What is `fake`?

Reply:
it's a copy of search input with event listener. real input is hidden inside popup by default on mobile resolution

Reply:
Suggest to rename into `fake-input` for example.